### ISSUE TITLE
MongoDB broken: Build failing

### ISSUE THREAD

**Issue by winfried-van-loon** [laradock/laradock#1018]
At the moment, the Travis CI build is failing with the following error:

```
No releases available for package "pecl.php.net/mongodb"
```

 The relevant part of the logs:

```
 ---> b94407fa0430
Removing intermediate container f886a28b418c
Step 44/115 : ARG INSTALL_MONGO=false
 ---> Running in 3d153b04a2af
 ---> 18462edbae49
Removing intermediate container 3d153b04a2af
Step 45/115 : ENV INSTALL_MONGO ${INSTALL_MONGO}
 ---> Running in 981b159ebac5
 ---> a0ed4cb65e30
Removing intermediate container 981b159ebac5
Step 46/115 : RUN if [ ${INSTALL_MONGO} = true ]; then     pecl install mongodb &&     echo "extension=mongodb.so" >> /etc/php/7.0/mods-available/mongodb.ini &&     ln -s /etc/php/7.0/mods-available/mongodb.ini /etc/php/7.0/cli/conf.d/30-mongodb.ini ;fi
 ---> Running in c023c3bc6729
No releases available for package "pecl.php.net/mongodb"
install failed
ERROR: Service 'workspace' failed to build: The command '/bin/sh -c if [ ${INSTALL_MONGO} = true ]; then     pecl install mongodb &&     echo "extension=mongodb.so" >> /etc/php/7.0/mods-available/mongodb.ini &&     ln -s /etc/php/7.0/mods-available/mongodb.ini /etc/php/7.0/cli/conf.d/30-mongodb.ini ;fi' returned a non-zero code: 1
The command "./travis-build.sh" exited with 1.
Done. Your build exited with 1.
```

The full logs can be found here (scroll down to the bottom to see the error):
https://travis-ci.org/laradock/laradock/jobs/242249521

---

*Update 1:*

This only happens for `PHP_VERSION=70 BUILD_SERVICE=workspace`.
The issue is probably somewhere here: https://github.com/laradock/laradock/blob/master/workspace/Dockerfile-70#L184

I've found [a post](https://stackoverflow.com/a/34486877/675951) that may suggest that adding `pecl channel-update pecl.php.net` may help, as we're missing it in our Dockerfile.

--- 

*Update 2:*

Same issue is present for php7.1, however it only appeared when the build for php7.0 was successful. Fixed: new build is testing now.

--- 

*Update 3:*

Using the `channel-update`, other non-related issues are appearing. For php5.6, redis is now failing to install:

```
Package "redis" Version "3.1.2" does not have REST xml available
```

Either we'll need to add the channel-update line there as well, or that "fix" does more harm than good.

**Comment by winfried-van-loon** [comment]
This seems to have been fixed in #1018 by #1035 and #1033. The error does not happen anymore. Good news! Thanks to @bestlong 👍

**Comment by lhq0826** [comment]
@winfried-van-loon hi, Same issue is present for php7.0, git commit '8a090f3e46f4d04f9111952b5d12f2e2c80ff7b9'
```
Step 19/69 : RUN if [ ${INSTALL_MONGO} = true ]; then     pecl install mongodb &&     docker-php-ext-enable mongodb ;fi
 ---> Running in b08a8ce25afe
No releases available for package "pecl.php.net/mongodb"
install failed
```
